src/utils/backwardsCompatibleContext.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+function backwardsCompatibleContext(Component, { deprecatedAPI: { contextTypes, childContextTypes }, newAPI: { contextType } }) {
+  if (typeof React.createContext === 'undefined') {
```

Comment:
⬜️ Add a comment explaining what React versions are expected to use the new context API (with link)